workspaces/quickstart/plugins/quickstart/src/components/QuickstartContent/QuickstartCtaLink.tsx

```diff
@@ -67,7 +75,12 @@ export const QuickstartCtaLink = ({ cta, onClick }: QuickstartCtaLinkProps) => {
       to={cta.link}
       onClick={onClick}
     >
-      {cta.text}
+      {cta.textKey
+        ? (() => {
+            const translation = t(cta.textKey as keyof typeof t, {});
+            return translation !== cta.textKey ? translation : cta.text;
+          })()
+        : cta.text}
```

Comment:
When doing the same code twice in the same component I would recommend to calculate this before the TSX code.